src/runtime-rs/crates/hypervisor/README.md

```diff
@@ -0,0 +1,79 @@
+# Multi-vmm support for runtime-rs
+Some key points for supporting multi-vmm in rust runtime.
+## 1. Hypervisor Config
+The diagram below gives an overvow for hypervisor config
+
+![hypervisor config](../../docs/images/hypervisor-config.svg)
```

Comment:
Images can be saved in `src/runtime-rs/docs/images/`, `runtime-rs` its own docs directory.

Reply:
fixed

Reply:
Oh, I missed the path..